java/runtime/src/main/resources/ray.default.conf

```diff
@@ -31,8 +31,11 @@ ray {
     resource-path: ""
   }
 
+  // The root dir of session.
+  session-dir: ""
+
   // Root dir of log files.
-  log-dir: /tmp/ray/logs
+  log-dir: ""
```

Comment:
should we document somewhere the priorities of these dirs? e.g. which will take effect when both are specified

Reply:
My bad. `session-dir` shouldn't be exposed to users. I'll remove it soon.

Reply:
My bad. session-dir shouldn't be exposed to users. I'll remove it soon.